pandas/util/testing.py

```diff
@@ -1578,7 +1578,18 @@ def skip_if_no_package(*args, **kwargs):
                   exc_failed_check=SkipTest,
                   *args, **kwargs)
 
-#
+def skip_if_no_package_deco(pkg_name, version=None, app='pandas'):
+    from nose import SkipTest
+
+    def deco(func):
+        @wraps(func)
+        def wrapper(*args, **kwargs):
+            package_check(pkg_name, version=version, app=app,
```

Comment:
I thought so too. There was `skip_if_no_package`, which is pretty much the body of the `wrapper`, but I couldn't use it as a decorator.